spec/README.md

```diff
@@ -15,6 +15,10 @@ It has four layers that are independent but designed to work together:
 -   [Envelope]: Handles authentication and serialization.
 -   [Bundle]: Defines a method of grouping multiple attestations together.
 
+The following diagram visualises the relationships between the envelope, statement and predicate layers.
+
+<img src="../images/specification.png" alt="Relationships between the envelope, statement and predicate layers" width="600">
```

Comment:
nit: Can the name be made more specific than 'specification'?  Perhaps 'envelope_relationships'

Reply:
done